### PR TITLE
Add new topic: scapy

### PR DESCRIPTION
**Pull request by Joe-Sin7h**
### Please confirm this pull request meets the following requirements:

- [X] I followed the contributing guidelines: <https://github.com/github/explore/blob/master/CONTRIBUTING.md>.
- [X] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [X] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [ ] Content (and my changes are in `index.md`)

> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [X] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/scapy` or `https://github.com/collections/scapy`)
- [X] My folder contains a `*.png` image (if applicable) and `index.md`
- [X] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/master/docs>

> Scapy has become very popular and it's a powerful interactive packet manipulation program, that's why I think this topic should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

**Comment by guedou** [comment]
Some information provided in this PR are incorrect. I will try to provide feedback ASAP.

**Comment by Joe-Sin7h** [comment]
> Some information provided in this PR are incorrect. I will try to provide feedback ASAP.

sure, let me know so I can fix them.

File: topics/scapy/index.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+aliases: scapy
+created_by: Philippe Biondi
+display_name: scapy
```
Comment (guedou) [inline topics/scapy/index.md:4]: The official writing is `Scapy`

File: topics/scapy/index.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+github_url: https://github.com/secdev/scapy
+logo: scapy.png
+released: April 19, 2021
```
Comment (guedou) [inline topics/scapy/index.md:7]: It was released on March 26, 2003.

File: topics/scapy/index.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+short_description: Scapy is a powerful interactive packet manipulation program.
```
Comment (guedou) [inline topics/scapy/index.md:8]: Could you use:
`Scapy is a Python-based interactive packet manipulation program & library. It upports Python 2 & Python 3.`

File: topics/scapy/index.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+topic: scapy
+related: scapy, python
```
Comment (guedou) [inline topics/scapy/index.md:10]: Is `scapy` useful in this context?

**Review by guedou — COMMENTED** [review]
Could you apply these changes?

**Comment by Joe-Sin7h** [comment]
> Could you apply these changes?

Thanks for reviewing them, I have now made the required changes.

**Review by guedou — COMMENTED** [review]
It looks OK. Thanks!

File: topics/scapy/index.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
+---
+aliases: scapy
```
Comment (kenyonj) [inline topics/scapy/index.md:2]: Since `scapy` is the topic name, it can't be listed as an alias as well.
```suggestion
```

**Review by kenyonj — APPROVED** [review]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

(I'll make the suggested change)

**Review by kenyonj — CHANGES_REQUESTED** [review]
Looks like there are still CI issues. Specifically:

```
topics::scapy topic#test_0019_has at most one image with the right name, type, and dimensions [/home/runner/work/explore/explore/test/topics_test.rb:229]:
topic images should be 288px wide.
Expected: 288
  Actual: 595
```

**Comment by Joe-Sin7h** [comment]
@kenyonj Thanks, I'll add a different image now.

**Review by kenyonj — APPROVED** [review]

**Comment by Joe-Sin7h** [comment]
@kenyonj can you merge it now :)
Thank you for reviewing.

**Pull request merged**